actors/migration/top.go

```diff
@@ -12,12 +13,30 @@ import (
 	"github.com/filecoin-project/specs-actors/actors/util/adt"
 	"github.com/ipfs/go-cid"
 	cbor "github.com/ipfs/go-ipld-cbor"
+	"golang.org/x/sync/semaphore"
 	"golang.org/x/xerrors"
 
 	"github.com/filecoin-project/specs-actors/v2/actors/builtin"
 	"github.com/filecoin-project/specs-actors/v2/actors/states"
 )
 
+var (
+	defaultMaxWorkers = 32
```

Comment:
This is a bit higher than I expected.

Reply:
I haven't swept it rigorously with the latest optimizations but the overhead from going too high seems very marginal.  I'll bring it down to 16 and maybe tweak after the migration is complete.